### PR TITLE
@kanabe => Revise animations

### PR DESCRIPTION
**Pull request by eessex**
Waypoints have mostly been taken out, and many css transitions have been replaced by pixel-by-pixel animations (for vertical lines in body) -- this is currently very laggy so today I'm going to investigate using GSAP instead of jQuery.  Let me know if you have ideas on this!

- Beginnings of responsive theming
- Support for additional writer fields
- Uses resized cloudfront images

### CODE REVIEW

File: apps/editorial_features/components/eoy/client.coffee [hunk 1]
```diff
@@ -28,57 +31,63 @@ module.exports.EoyView = class EoyView extends Backbone.View
...
-      $('.scroller__items section[data-section="0"]').attr('data-state', 'open').height(@firstHeight)
-    #else if scrollTop > @windowHeight
+      $('.scroller__items section[data-section="0"]').attr('data-state', 'open').height(@containerHeight)
+    #else if scrollTop > @windowPosition
       #downscrolling
     #else
       #upscrolling
     @doSlider(scrollTop)
     if scrollTop >= @getScrollZones()[6]
+      # throttled = _.throttle(@animateBody, 100)
```
Comment (kanaabe) [inline apps/editorial_features/components/eoy/client.coffee]: Would it be ok to remove this commented out bits of code? I see a few in the Stylus sheets as well.

**Comment by kanaabe** [comment]
Minor comments inline. Let me know what you think about GSAP - I've never used it before but I see they have an NPM package which is great. I guess you can also install it as a jQuery plugin which could be useful for our semi one-off situation. We can discuss.

We should also start putting in some tests for the client stuff if you're feeling pretty set on the pixel by pixel animation. I know you've been holding off until you sort out the animation/waypoint stuff but it feels like it's in a good place now. 👍

**Comment by eessex** [comment]
@kanaabe - made some of these changes and I've just started a new branch for testing. Having some trouble getting this set up, maybe we can pair on this tomorrow if you have time?

**Comment by kanaabe** [comment]
Yep, sure. Mergin this one for now!

**Pull request merged**